net-p2p/qbittorrent/qbittorrent-5.0.0.ebuild

```diff
@@ -13,45 +13,32 @@ if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 else
 	SRC_URI="
-		https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz
-		verify-sig? ( https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz.asc )
+		https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz
+		verify-sig? ( https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz.asc )
```

Comment:
I think that most of the time packages use the "short url" actually. Both are stable urls.

Reply:
The motivation for the RedirectedUrl warning type is I think for cases where projects migrate their layout and leave temporary redirects. We want to make sure we are using the canonical location so that things don't break in the future.

Sourceforge is a bit weird in this sense. The urls are actually their (badly documented) mapping of download urls to internal storage hierarchy. Most packages in Gentoo using sourceforge, as far as I can tell, just ignore the existence of that warning.